backend/package.json

```diff
@@ -18,7 +18,8 @@
     "jade": "^1.11.0",
     "jsonwebtoken": "^8.5.1",
     "mssql": "^7.3.0",
-    "sequelize": "^5.22.3"
+    "sequelize": "^5.22.3",
+    "tedious": "^14.1.0"
```

Comment:
Remove Tedious if not utilized

Reply:
On my side , node crashes without tedious